index.js

```diff
@@ -64,27 +76,8 @@ class PhotoViewManager {
     return scale;
   }
 
-  imageTouchHandler(e) {
```

Comment:
Why these got removed ? I remember these are required to fix an issue with Pan and Touchswipe

//cc @izebb